authy-offline.py

```diff
@@ -85,18 +85,41 @@ def parseXML(dataFile):
         border=2,
     )
 
+    known_keys = {
+        # fields we use:
+        "accountType", "decryptedSecret", "digits", "name", "originalIssuer", "originalName", "timestamp",
+        # fields we know we don't need:
+        "encryptedSecret", "salt", "key_derivation_iterations", "upload_state", "hidden", "id", "isNew", "logo",
```

Comment:
Locally, it seems `key_derivation_iterations` is usually 100000, `upload_state`, `hidden`, `isNew` are all details about Authy's state for the account, `id` is some internal ID, and `logo` seems to be some short name for the logo Authy displays (there doesn't seem to be a straightforward way to use this with pyotp: it expects a URL instead of a short string).